{
turnRate    : 10, #deg per second
thrustScale : 1,  #scale of thrust to speed conversion

#ocean current simulation 
currentSpeed    : 0, #meter per second
currentYaw      : 0, #radian

#vehicle start location, lat lon takes precedence
startX_lat      : 39.3265546541211108,
startY_lon      : -76.6221714019775391,

startX          : 0,
startY          : 0

}